interactions/client/models/component.py

```diff
@@ -286,26 +236,6 @@ class ActionRow(ComponentMixin):
     type: ComponentType = field(ComponentType, default=ComponentType.ACTION_ROW)
     components: Optional[List[Component]] = field(converter=convert_list(Component), default=None)
 
-    def __attrs_post_init__(self) -> None:
-        for component in self.components:
-            if isinstance(component, SelectMenu):
-                component._json["options"] = (
-                    [
-                        option._json if isinstance(option, SelectOption) else option
-                        for option in component._json["options"]
-                    ]
-                    if component._json.get("options")
-                    else []
-                )
-        self.components = (
-            [Component(**component._json) for component in self.components]
-            if self._json.get("components")
-            else None
-        )
-        self._json.update({"type": self.type.value})
-        if self._json.get("components"):
-            self._json["components"] = [component._json for component in self.components]
-
     @classmethod
     def new(cls, *components: Union[Button, SelectMenu, TextInput]) -> "ActionRow":
```

Comment:
```suggestion
    def new(cls, *components: Union[Button, SelectMenu, TextInput]) -> List["ActionRow"]:
```

If you change the method you can fix the rtype aswell